### PR TITLE
Create and Publish MiningStatistics SignalR event

### PR DESCRIPTION
**Pull request by SatyaKarki**

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.PoA/Events/MiningStatisticsEvent.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.Text;
+using Stratis.Bitcoin.EventBus;
+
+namespace Stratis.Bitcoin.Features.PoA.Events
+{
+    public class MiningStatisticsEvent : EventBase
+    {
+        public MiningStatisticsModel MiningStatistics { get; }
+        public readonly int FederationMemberSize;
+        public MiningStatisticsEvent(MiningStatisticsModel miningStatistics, int federationMemberSize)
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.PoA/Events/MiningStatisticsEvent.cs:12]: Please add a emptuy line above the constructor (code style)

File: src/Stratis.Bitcoin.Features.PoA/Events/MiningStatisticsEvent.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Stratis.Bitcoin.EventBus;
+
+namespace Stratis.Bitcoin.Features.PoA.Events
+{
+    public class MiningStatisticsEvent : EventBase
+    {
+        public MiningStatisticsModel MiningStatistics { get; }
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.PoA/Events/MiningStatisticsEvent.cs:10]: Make this `readonly` too.

File: src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs [hunk 3]
```diff
@@ -299,6 +305,11 @@ private void GatherMiningStatistics()
 
             this.miningStatistics.MinerHits = hitCount;
 
+            if (this.signals != null)
```
Comment (fassadlr) [inline src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs:308]: Any reason for the null check?

Comment (SatyaKarki) [reply src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs:308]: If there is no signals, I guess it may throw error publishing the event.

**Review by fassadlr — APPROVED** [review]

**Pull request merged**